sdk/search/azure-search/src/main/java/com/azure/search/AutocompletePagedResponse.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search;
+
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.rest.Page;
+import com.azure.core.http.rest.PagedResponseBase;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.search.models.AutocompleteItem;
+import com.azure.search.models.AutocompleteResult;
+
+import java.util.stream.Collectors;
+
+/**
+ * Represents an HTTP response from the autocomplete API request
+ * that contains a list of items deserialized into a {@link Page}.
+ */
+public class AutocompletePagedResponse extends PagedResponseBase<String, AutocompleteItem> {
+
+    /**
+     * Constructor
+     * @param autocompleteResponse an http response with the results
+     */
+    public AutocompletePagedResponse(SimpleResponse<AutocompleteResult> autocompleteResponse) {
```

Comment:
I've added the missing `@search.coverage` annotation to the service and the change has been deployed, so we'll need to add a coverage field and accessors here in a subsequent PR.